django_app/tests/conftest.py

```diff
@@ -53,8 +60,15 @@ def jemima_puddleduck():
 
 
 @pytest.fixture()
-def mrs_tiggywinkle():
-    return User.objects.create_user(email="mrs.tiggywinkle@example.com")
+def mrs_tiggywinkle(business_unit: BusinessUnit) -> User:
```

Comment:
Is it worth renaming this to something like `user_with_demographic_info` for test readability?